README.md

```diff
@@ -29,15 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+> `map` returns values and will return a new array vs. `forEach` doesn't return anything, it calls a provided function on each element in the array
 2. What is the difference between a function and a method?
-
+> They are both functions, a method is property of a function.
```

Comment:
Yes, correct, they are both functions. 

But a method is a function nested in an object.